torch/_inductor/config.py

```diff
@@ -79,6 +79,9 @@
 # enable pattern match with batch fusion (using torch op)
 batch_fusion = True
 
+# enable post-grad pattern match with batch fusion (using torch op)
+post_grad_batch_fusion = False
+
```

Comment:
I don't think we have to add a separate flag for this, we already have ```batch_fusion``` at L80.

Reply:
I thought `batch_fusion` at L80 is pre_grad? I am thinking it would still be useful to make them separate config flags for debugging purpose.